notebooks/official/generative_ai/text_embedding_api_semantic_search_with_scann.ipynb

```diff
@@ -0,0 +1,1153 @@
+{
```

Comment:
<p>Change Pre-requisites to match the template for Objective:</p><p> </p><p>e.g., uses these services, and these are the steps.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>